app/commands/github/issue/open.rb

```diff
@@ -6,6 +6,8 @@ class Open
       initialize_with :repo, :title, :body
 
       def call
+        return if Rails.env.development?
```

Comment:
I feel like we want this to be `return unless Rails.env.production?` but then I imagine that will break the tests?

Reply:
It will